ES/ESmy015.xml

```diff
@@ -0,0 +1,513 @@
+                         <incipit xml:lang="gez"><hi rend="rubric">አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡</hi> ኅቡረ፡ ህላዌ፡ ሥሉስ፤ በተገጽዎ፡ አብ፡
+                            ወወልድ፤ ወመንፈስ፡ ቅዱስ፡ በጽማሬ፡ ሥሉስ፡ በ<hi rend="rubric">ሥላሴ፡</hi> ጽሙር፡ አብ፡ በአካሉ፨</incipit>
+                      </msItem>
+
+                 <msItem xml:id="ms_i1.5">
+                      <locus from="19vb" to="24rb"></locus>
+                    <title ref="LIT2296Sayfas#Thursday">Prayer for Thursday</title>
+                         <incipit xml:lang="gez">አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ በግፃዌ፡ ሥሉስ፡ በሥላሴ፡ ዋህድ፡
+                            አምላክ፡ ፫አዕማደ፡ ብርሃን፡ ፩አምጣኖሙ፡ ው፩ስኖሙ፡ ፫መባርቅት፡ ፩ንጥረ፡ መለኮት፡</incipit>
+                      </msItem>
+
+                   <msItem xml:id="ms_i1.6">
+                      <locus from="24rb" to="29vb"></locus>
+                         <title ref="LIT2296Sayfas#Friday">Prayer for Friday</title>
+                         <incipit xml:lang="gez"><hi rend="rubric">አይ፡ ልሳን፡ ወአይ<supplied reason="omitted" resp="DN">፡</supplied>
+                          ከናፍር፡ ዘይክል፡ ነቢበ፡ ዕበየ፡</hi> ክብሩ፡ <hi rend="rubric">ለሥላሴ፡</hi> ቅዱስ፡ አይ፡ ልቡና፡</incipit>
+                      </msItem>
+
+                   <msItem xml:id="ms_i1.7">
+                      <locus from="30ra" to="33ra"></locus>
+                    <title ref="LIT2296Sayfas#Saturday">Prayer for Saturday</title>
+                         <incipit xml:lang="gez"><hi rend="rubric">ሰላም፡ ለፍቅረ፡ መለኮ</hi>ት፡ በትሥብዕቱ፡ ዘጾመ፡ እንበይነ፡
+                            ዚአነ፡ አርአያሁ፨ ከመ፡ የሀበነ፡ ሰላም፡ ለፍቅረ፡ መለኮትከ፨</incipit>
+
+                      </msItem>
+
+                   <msItem xml:id="ms_i1.8">
+                      <locus from="33ra" to="36rb"/>
+                         <title ref="LIT2296Sayfas#Sunday">Prayer for Sunday</title>
+                         <incipit xml:lang="gez"><hi rend="rubric">አኃሥሥ፡ ዘእምኀቤከ፡ ኦቀመረ፡ ጽድቅ፡ አኃሥሥ፡ ጥ</hi>በበ፡ እምኔከ፨ ኦሙዳየ፡ ጥበብ፡</incipit>
+                      </msItem>
+
+                      <msItem xml:id="ms_i1.9">
+                         <locus from="36rb" to="37rb"/>
+                            <title ref="LIT2296Sayfas#Daily">Reading for each day</title>
+                            <incipit xml:lang="gez">
+                              <del rend="effaced"/>አብ፤ ወላዲ፤ ወልድ፤ ቅዱስ፤ ይትባረክ፤ እግዚአብሔር፤ ወልድ፤
+                              ወ<supplied reason="omitted" resp="DN">ል</supplied>ደ፤
+                              አብ፤ ዋህድ፤ ይትባረክ፤ እግዚአብሔር፤ ጸራቅሊቶስ፡ መንፈሰ፡ አብ፡ ወወልድ፡ ወመንፈሰ፡ አብ፡ አ<pb n="36vb"/>ምላክ፡ ውእቱ፡</incipit>
+                         </msItem>
+
+                      <msItem xml:id="ms_i1.10">
+                         <locus from="37rb" to="43ra">Seven Miracles of the Trinity</locus>
+                         <title ref="LIT2296Sayfas#Miracles"/>
+
+                         <msItem xml:id="ms_i1.10.1">
+                            <locus from="37rb" to="37vb"/>
+                            <title ref="LIT2296Sayfas#Miracles">Miracle 1. The God Father announces the future Reign of the Trinity</title>
```

Comment:
either individual IDs should be introduced for the miracles, or here the ref (repeating the upper order ref) should be taken out, as it is also not present in Miracle 2 etc